tests/test_vrf.rb

```diff
@@ -107,6 +107,59 @@ def test_description
     v.destroy
   end
 
+  def test_mhost
+    v = Vrf.new('test_mhost')
+    t_intf = 'Loopback100'
+    if platform == :nexus
```

Comment:
https://github.com/cisco/cisco-network-node-utils/pull/279, added a new method `validate_property_excluded?()` that can be used instead of checking against a specific platform or os.

```
if validate_property_excluded?('vrf', 'mhost_ipv4')
  # assert_nil and assert_raises checks here
end
```